client/product_details/DefaultView.jsx

```diff
@@ -1,18 +1,30 @@
 import React from 'react';
+import ThumbnailList from './ThumbnailList.jsx';
```

Comment:
Do you need this import statement in this file? Doesn't look like you're rendering this component here but maybe you will in the future - just checking!

Reply:
I forgot that in there. I'm not rendering it in this component anymore. I will update this on the next PR.